src/date-picker/__tests__/date-picker-calendar.test.tsx

```diff
@@ -40,7 +40,9 @@ describe('Date picker calendar', () => {
   }
 
   const findFocusedDay = (wrapper: DatePickerWrapper) => {
-    return wrapper.findCalendar()!.find(`.${calendarStyles['calendar-day']}[tabIndex="0"]`);
+    return wrapper
+      .findCalendar()!
+      .find(`.${calendarStyles['calendar-day']}[tabIndex="0"] :not(.${calendarStyles['visually-hidden']})`);
```

Comment:
Probably can update this one with:

```
wrapper
  .findCalendar()!
  .findComponent(`.${calendarStyles['calendar-day']}[tabIndex="0"]`, CalendarDayWrapper)!
  .findLabel()
```
That would ensure regression for the new `findLabel` util.

Reply:
Noticed that findLabel is already used below - resolving conversation